openlcsd/flow/tasks.py

```diff
@@ -841,17 +914,26 @@ def fork_components_imports(context, engine):
     Fork components tasks
     """
     components = context.get('components')
-    srpm_nvr_list = \
-        get_nvr_list_from_components(components, 'SRPM')
-    # Fork srpm component tasks
+    srpm_nvr_list = get_nvr_list_from_components(components, 'SRPM')
+
+    # Fork source RPM component tasks.
     if srpm_nvr_list:
-        import_types_components(context, engine, srpm_nvr_list,
-                                context.get('srpm_dir'), 'SRPM')
-    # Fork container-compnent tasks with the misc metadata files
+        import_types_components(
+            context, engine, srpm_nvr_list, context.get('srpm_dir'), 'SRPM')
+
+    # Fork container-component tasks with the misc metadata files.
     if components.get('CONTAINER_IMAGE'):
         import_types_components(context, engine, [context.get('package_nvr')],
                                 context.get('misc_dir'), 'CONTAINER_IMAGE')
 
+    # Fork remote source component tasks.
+    comp_types = ['GOMOD', 'GOLANG', 'NPM', 'YARN', 'PIP', 'PYPI']
```

Comment:
GOMOD==GOLANG and PIP==PYPI, why list them both here?

Reply:
This is incorrect,updated